libs/blocks/aside/aside.js

```diff
@@ -46,11 +47,70 @@ function decorateStaticLinks(el) {
   textLinks.forEach((link) => { link.classList.add('static'); });
 }
 
+function decorateMedia(el) {
+  if (!(el.classList.contains('medium') || el.classList.contains('large'))) return;
+  const allMedia = Array.from(el.querySelectorAll('div > p video, div > p picture'));
+  allMedia.some((media) => {
```

Comment:
Nit, use array spread syntax:
```suggestion
  const allMedia = el.querySelectorAll('div > p video, div > p picture');
  [...allMedia].some((media) => {
```